src/PKSim.Core/Commands/EditParameterCommand.cs

```diff
@@ -104,6 +111,64 @@ private void setBuildingBlockAlteredFlag(IExecutionContext context)
          }
       }
 
+      protected void SaveValueOriginFor(IParameter parameter)
+      {
+         _oldValueOrigin = parameter.ValueOrigin.Clone();
+      }
+
+      protected virtual void UpdateParameter(IExecutionContext context)
+      {
+         var bbParameter = OriginParameterFor(_parameter, context);
+         UpdateParameter(_parameter, context);
+         UpdateParameter(bbParameter, context);
+         UpdateValueOriginInParameters(_parameter, bbParameter);
```

Comment:
I am always updating teh VAlueOrigin in both parmaeter in corresponding bbParameter in Simulation